src/components/Table/Table.module.css

```diff
@@ -38,3 +38,8 @@
     padding-bottom: var(--column-padding) !important;
   }
 }
+
+.action {
+  margin: calc(var(--spacing-margin-md, 12) * -1px);
```

Comment:
Why -1? 😧

Reply:
We need to constrain the button with a negative margin to hide the ripple effect around the button

Reply:
lo mettiamo commentato sopra? così lo ricordiamo tra X o Comunque è visibile alle persone che guardano questo codice?